src/routes/product/[id([0-9]+)]/index.svelte

```diff
@@ -0,0 +1,210 @@
+<script context="module">
```

Comment:
Too much for a page. It's better being split

Reply:
How would you suggest splitting it? I guess we could put the purchase modal in a container, but that's about it